app/Console/Kernel.php

```diff
@@ -31,6 +32,7 @@ protected function schedule(Schedule $schedule): void
 
         $schedule->job(new WeeklyAttendance())->weekly()->sundays()->at('11:00');
         $schedule->job(new DailyDuesSummary())->daily()->at('11:00');
+        $schedule->job(new NoAttendanceJediPush())->daily()->at('5:00');
```

Comment:
I would rather send emails around 10am for better visibility.

Reply:
Will try removing someone from the billing group while they're signed into a machine before pushing that back.

Reply:
As discussed in Slack, removing someone from a billing group while they're signed in does not sign them out. They are unable to sign into the SUMS kiosk to sign out of the machine, but other people will be around who can.